### PR TITLE
Add 8ball plugin

### PR DESCRIPTION
**Pull request by gooca**
#4 head

### CODE REVIEW

File: src/app/plugins/8ball.plugin.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args?: string[]) {
+    const response = this._responses[Math.floor(Math.random() * this._responses.length)];
+    this._embed.setColor('#0099ff').setTitle(response);
+    this._embed.setAuthor(
+      'The magic 8ball says...',
+      'https://cdn.emojidex.com/emoji/seal/8ball.png'
```
Comment (c650) [inline src/app/plugins/8ball.plugin.ts]: should be class constant

File: src/bootstrap/plugin.loader.ts [hunk 2]
```diff
@@ -14,6 +15,7 @@ const PluginStore: { [pluginName: string]: any } = {
   dog: DogPlugin,
   users: UserCountPlugin,
   garage: GaragePlugin,
+  "8ball": EightBallPlugin,
```
Comment (c650) [inline src/bootstrap/plugin.loader.ts]: inconsistent !!

File: src/app/plugins/8ball.plugin.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args?: string[]) {
+    const response = this._responses[Math.floor(Math.random() * this._responses.length)];
+    this._embed.setColor('#0099ff').setTitle(response);
+    this._embed.setAuthor(
+      "The magic 8ball says...",
+      "https://cdn.emojidex.com/emoji/seal/8ball.png"
```
Comment (c650) [inline src/app/plugins/8ball.plugin.ts]: make class constant

File: src/app/plugins/8ball.plugin.ts (hunk 1 above)
Comment (gooca) [reply src/app/plugins/8ball.plugin.ts]: whats a class constant

**Pull request merged**